samcli/local/apigw/local_apigw_service.py

```diff
@@ -230,6 +235,17 @@ def _parse_lambda_output(lambda_output, binary_types, flask_request):
 
         return status_code, headers, body
 
+    @staticmethod
+    def _invalid_apig_response(output):
+        allowable = [
+            "statusCode",
+            "body",
+            "headers",
+            "isBase64Encoded"
+        ]
+        invalid_keys = set(output) - set(allowable)
+        return invalid_keys != set()
```

Comment:
`return bool(invalid_keys)` is sufficient! In Python empty list/set is considered Falsy. The `!=` syntax is not too Pythonic ;)

Reply:
Same story, tests still pass after this change, just merged.